fchost/fchost.go

```diff
@@ -19,8 +22,9 @@ var (
 
 // FilecoinHost is a libp2p host connected to the FC network
 type FilecoinHost struct {
-	host.Host
-	dht *dht.IpfsDHT
+	ping *ping.PingService
+	h    host.Host
+	dht  *dht.IpfsDHT
```

Comment:
`host.Host` isn't embedded anymore, so less power for users of `FilecoinHost`.
Preferred this way and implement as public methods whatever they need.